backend/app/tests/utils/openai.py

```
@@ -0,0 +1,39 @@
+from typing import Optional
+import time
+
+from openai.types.beta import Assistant as OpenAIAssistant
+from openai.types.beta.assistant import ToolResources, ToolResourcesFileSearch
+from openai.types.beta.assistant_tool import FileSearchTool
+from openai.types.beta.file_search_tool import FileSearch
+
+
+def mock_openai_assistant(
+    assistant_id: str = "assistant_mock",
+    vector_store_ids: Optional[list[str]] = ["vs_1", "vs_2"],
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)